### DISCUSSION TITLE
Masque not Working Properly with Custom Texts

### DISCUSSION THREAD

**Discussion by NavaresEU** [SFX-WoW/Masque#183] [Issues] [answered]
![desktop screenshot 2018 08 12 - 23 09 26 02](https://user-images.githubusercontent.com/42326632/44007097-db6bd692-9e8f-11e8-87bd-d768527f8ae9.png)

Here is the issue: When using WAs as showed in the screenshot inside a dynamic group, the first aura of the group will always have some weird issues with Masque: like the custom text going behind the border while it is suposed to stay on top, sometimes the borders become bright as well.

I've been using theses WA for a while and i'm seeing this issue for the first time after having updated WA to 2.7, the issue only occcur with Masque enabled.

**Comment by StormFX** [comment]
This may be related to #19, if WA is using the Cooldown frame for its text. If not, then it's something that the authors of WeakAuras will have to deal with since Masque only skins the default button layers.

**Comment by NavaresEU** [comment]
Thank you for your response, actually the issue might nor even come frome the WA update because I tried coming back to the previous WA version and I'm still having the issue.

Do you have any idea of how I could fix this ? I really don't understand where this issue is coming from, i've been using theses WA for a while now and i've never seen this

**Comment by StormFX** [comment]
I'll look into it and see if it's related to the aforementioned issue. In the meantime, see if WA has an option for the custom text that allows you set its strata.

**Comment by NavaresEU** [comment, marked as answer]
It is definetely a WA issue and not a Masque issue, I will try to fix it with them. Thanks
